less_users/tests/test_get_points.py

```diff
@@ -0,0 +1,88 @@
+from freezegun import freeze_time
+
+from datetime import date, timedelta
+
+from less_users.models import UserChallenge, Log
+
+
+def test_get_points_for_userchallenge_before_first_log(user, challenge_3_month):
+    userchallenge = UserChallenge.objects.create(user=user, challenge=challenge_3_month)
+
+    assert userchallenge.get_points == challenge_3_month.points
```

Comment:
Warto by dodać assert, że żadne logi nie istnieją.

Reply:
np.`assert not userchallenge.log_set.exists()` ?